susemanager-utils/susemanager-sls/src/modules/kiwi_info.py

```diff
@@ -478,23 +458,21 @@ def inspect_bundles(dest, basename):
         if match:
             res1 = match.groupdict()
             sha256_file = f
-            # pylint: disable-next=undefined-variable
             sha256_str = __salt__["cp.get_file_str"](os.path.join(dest, sha256_file))
-            pattern2 = re.compile(r"^(?P<hash>[0-9a-f]+)\s+(?P<filename>.*)\s*$")
+            pattern2 = re.compile(r"^(?P<hash>[0-9a-f]+)\s+(?P<filename>\S.*)$")
             match = pattern2.match(sha256_str)
             if match:
                 d = match.groupdict()
-                # pylint: disable-next=consider-using-f-string
-                d["hash"] = "sha256:{0}".format(d["hash"])
+                d["filename"] = d["filename"].strip()
+                d["hash"] = f'sha256:{d["hash"]}'
```

Comment:
Is this safe to use now?

Reply:
Just double checking there will be no surprise with old python version. I remember I had to revert such changes in saltboot because of older python version some year back.